index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <header id="header">
+      
+        <img width="300px" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="picture" id="header-img">
+        
+        <nav id="nav-bar">
+          
+            <a class="nav-link" href="#features">Features</a></li>
+            <a class="nav-link" href="#howitworks">How it Works</a></li>
+            <a class="nav-link"  href="#pricing">Pricing</a></li>
```

Comment:
Please wrap your links in a ul tag. wrapping them in the tag will help you position or style them better